packages/zosfiles/__tests__/__system__/api/methods/rename/Rename.system.test.ts

```diff
@@ -139,7 +137,7 @@ describe("Rename", () => {
                 let response;
 
                 try {
-                    response = await Rename.dataSet(REAL_SESSION, beforeDSName, "");
+                    response = await Rename.dataSet(REAL_SESSION, beforeDataSetName, "");
```

Comment:
What about `"  "` or `"  some.dataset"`? (Spaces that could be trimmed)

Reply:
Since the API now uses `IMperativeExpect.toBeDefinedAndNonBlank` it's safe to assume there will be no blank dataset names.
Would be nice if we could ad a unit test reflecting this change : )